src/highlight/HighlightAnnotation.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import * as ReactRedux from 'react-redux';
+import classNames from 'classnames';
+import noop from 'lodash/noop';
+import { getIsCurrentFileVersion } from '../store';
```

Comment:
For import statements, I'm guessing that there isn't a general convention for separating out different types of imports?
Ie:

library imports 

bui imports

absolute imports

etc

Reply:
We typically follow a convention of: absolute imports, relative imports, scss with each type alphabetized